get_data/coronadata.py

```diff
@@ -1,20 +1,26 @@
 from datetime import date
 import pandas as pd
 import requests
+import os
+from importData import call_macro
```

Comment:
Can we make sure these files/directories are consistent with naming conventions please?